enclave-tls/src/lib/tls_wrappers/null/receive.c

```diff
@@ -0,0 +1,21 @@
+#include <unistd.h>
+
+#include <enclave-tls/log.h>
+#include <enclave-tls/err.h>
+#include <enclave-tls/tls_wrapper.h>
+
+/* *INDENT-OFF* */
+tls_wrapper_err_t null_receive(tls_wrapper_ctx_t *ctx, void *buf,
+			       size_t *buf_size)
+{
+	ETLS_DEBUG("tls_wrapper_null receive() called\n");
+
+	*buf_size = read(ctx->fd, buf, *buf_size);
+	if (*buf_size < 0) {
```

Comment:
*buf_size never return negative value. This is a **BUG**. Refer to the man page of read() to correct this fatal.